src/_sass/_colors.scss.template

```diff
@@ -62,6 +63,7 @@ $checked_overlay_color:     gtkalpha(currentColor, 0.24);
 $primary_color:            %SEL_BG%;
 $alt_primary_color:        %SEL_BG2%;
 $selected_overlay_color:    rgba($primary_color, 0.32);
+$selected_checkradio_color:  %ACCENT_BG%;
```

Comment:
Why is this needed?

Reply:
see commit message :-)

this stopped working after some recent Materia changes

Reply:
because the use of `-gtk-recolor()` it stopped working, to be precise

Reply:
Ah, sorry. I was going to tell it to you, but I'm going to drop the (teal) accent/secondary color on the next release (in order to follow the Material Design guidelines more accurately). So the blue `check` and `radio` are intentional.

Would you mind removing `ACCENT_BG` and the option in Oomox eventually? IIRC, that is only used for Materia-based theme. I think specifying only `SEL_BG` as an accent color is also good thing for compatibility with other oomox themes.

Reply:
it's used not only in Materia theme, but it's possible to remove it only for Materia theme[1] and make it use some other color instead[2]

[1] https://github.com/themix-project/oomox/blob/master/plugins/theme_materia/oomox_plugin.py#L44
[2] https://github.com/themix-project/oomox/blob/master/plugins/theme_materia/oomox_plugin.py#L80-L87

Reply:
so, should i remove Accent option from Materia plugin completely?

Reply:
> it's used not only in Materia theme

Ah, right, sorry for my misunderstanding.

> so, should i remove Accent option from Materia plugin completely?

Hmm, yes, please. :pray: 

(Considering the Material Design guidelines, I think it's better. Also since changing the color of specific SVG assets/ID to `ACCENT_BG` is quite difficult now.)